tools/CompileReleaseNotes.py

```diff
@@ -329,9 +345,11 @@ def format_list_of_prs(prs):
             pr.author = pr_gh.user.login
             # Add missing metadata to PR
             pr.url = pr_gh.html_url
-            # Add group information to PR
+            # Add group information to PR. Iterate over groups in reverse order
+            # so that PRs with both, say, "bugfix" and "cli/pybindings" labels
+            # are listed under "bugfix" rather than "cli/pybindings".
```

Comment:
Would it be easy to just include it twice? Once under `bugfix` and once under `cli/pybindings`? (and more generally for many groups). That way we don't have to assign a kind of priority to a label? If not then don't worry about it